.hooks/pre-commit

```diff
@@ -62,11 +62,14 @@ ui_lint() {
 
 backend_lint() {
   # Silently succeed if no changes staged for Go code files.
-  if git diff --name-only --cached --exit-code -- '*.go'; then
+  staged=$(git diff --name-only --cached --exit-code -- '*.go')
+  ret=$?
+  if [ $ret -eq 0 ]; then
```

Comment:
Since this is `bash`, do we want to do `(( ret == 0 ))`?

Reply:
https://tldp.org/LDP/abs/html/dblparens.html -- I think for this, since this is the only numeric manipulation we're performing, we can keep `-eq`. :-)

Reply:
Nice, thanks for the insight! I'll keep that in mind for future reference. As you say, this is probably fine for now, so I'm going to go ahead and mark this as resolved 😄